src/unix/fs.c

```diff
@@ -739,6 +734,36 @@ static int uv__fs_fstat(int fd, uv_stat_t *buf) {
 }
 
 
+typedef ssize_t (*uv__fs_buf_iter_yield)(uv_fs_t* req);
+static ssize_t uv__fs_buf_iter(uv_fs_t* req, uv__fs_buf_iter_yield yield) {
+  unsigned int iovmax=uv__getiovmax();
+  unsigned int nbufs=req->nbufs;
+  uv_buf_t *bufs=req->bufs;
+  ssize_t total=0;
+  ssize_t result;
+
+  while(nbufs) {
+    req->nbufs=nbufs;
+    if(req->nbufs>iovmax) req->nbufs=iovmax;
+
+    result=yield(req);
+    if(result<=0) {
+      total=result;
+      break;
+    }
+
+    if(req->off>0) req->off+=result;
+    req->bufs+=req->nbufs;
+    nbufs-=req->nbufs;
+    total+=result;
+  }
+
+  if (bufs != req->bufsml)
+    uv__free(bufs);
+  return total;
+}
```

Comment:
Bunch of style issues in this function.  Necessary changes:

* Asterisk leans to the left (`uv_buf_t* bufs`)
* space between keywords and parens (`while (nbufs) {`.)
* spaces around `=`, `<=`, etc.
* libuv code tends to split out declaration and assignment of locals.